packages/teleport/src/services/user/makeAcl.ts

```diff
@@ -49,8 +47,7 @@ export default function makeAcl(json): Acl {
   // Behaves like clipboardSharingEnabled, see
   // https://github.com/gravitational/teleport/pull/12684#issue-1237830087
   const directorySharingEnabled =
-    (json.directorySharing !== undefined ? json.directorySharing : true) &&
-    cfg.enableDirectorySharing;
+    json.directorySharing !== undefined ? json.directorySharing : true;
```

Comment:
This handles undefined but not null. Is this intentional?

Reply:
`json` comes [directly from the backend](https://github.com/gravitational/teleport/blob/bc19ff0e439b09f62ec86df821d69a83d285df0a/lib/web/ui/usercontext.go#L91) and is converted to a javascript object, so technically it should always be defined. This logic is just me being overly cautious (similar to the variables above), since technically `json` is type `any` and so according to Typescript, `json.directorySharing` might be `undefined`.

Reply:
I would have just done

```
const directorySharingEnabled = !!json.directorySharing
```

but I suppose it doesn't really matter.